generators/assets/imageGenerator.js

```diff
@@ -8,69 +12,233 @@ Promise.promisifyAll(gm.prototype);
 
 /* eslint-disable no-multi-spaces */
 const iosIconSizes = [
-  { size: 20, multipliers: [1, 2, 3] },
-  { size: 29, multipliers: [1, 2, 3] },
-  { size: 40, multipliers: [1, 2, 3] },
-  { size: 50, multipliers: [1, 2] },
-  { size: 57, multipliers: [1, 2] },
-  { size: 60, multipliers: [2, 3] },
-  { size: 72, multipliers: [1, 2] },
-  { size: 76, multipliers: [1, 2] },
-  { size: 83.5, multipliers: [2] },
-  { size: 1024, multipliers: [1] },
+  {
```

Comment:
Seems like changes based on your IDE autoformat configuration, are they in sync with our prettier config?

Reply:
Oups, you're right, it seems like I am not in sync with your prettier config.
I will look at this shortly.

Reply:
Hi, I just push changes in that way + a side effect fix